tests/e2e/specs/ui/scenarios/popup.js

```diff
@@ -39,12 +39,13 @@ describe('Scenarios: shows correct popup value', () => {
         const text = $tooltipEl.text()
 
         const variant = text.split(' - ')[1]
-        const number = variant.split('.')[0]
+        const label = variant.split('.')[0]
```

Comment:
These changes seem to work for know, needs a bit more work to make it more robust and reliable.